jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -104,11 +107,34 @@ private static void assertExtractionPathsUnordered(
   @Mock private AuthProperty authProperty;
   @Mock private Consumer<LogEvent> logger;
 
+  /** Configuration for {@code platform} parameter. */
+  public static class PlatformParametersTest implements PlatformConfiguration {
+    @Nullable private String os;
+    @Nullable private String architecture;
+
+    PlatformParametersTest(String architecture, String os) {
```

Comment:
```suggestion
    PlatformParametersTest(@Nullable String architecture, @Nullable String os) {
```

Although we don't strictly enforce using `@Nullable annotation in test classes, let's annotate these properly since you used `@Nullable` for the fields.